### PR TITLE
added new wix-scopes plugin

### PR DESCRIPTION
**Pull request by Galcarmi**

### CODE REVIEW

File: wix-scopes/src/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+const FRONTEND_SCOPE = 'frontend'
+const BACKEND_SCOPE = 'backend'
+const UNIVERSAL_SCOPES = [FRONTEND_SCOPE, BACKEND_SCOPE]
+
+function extractScopesFromTag(tag){
+  if(!tag || !tag.value){
+    return UNIVERSAL_SCOPES
+  }
+
+  const isScropesTagValid = tag.value.match(/^\[[a-zA-Z,]*\]$/)
```
Comment (nemni8) [inline wix-scopes/src/index.js:11]: please change the regex to match only if frontend, backend or both

File: wix-scopes/src/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  if(!isScropesTagValid){
+    throw new Error(`scopes tag ${tag.value} is not valid`)
```
Comment (nemni8) [inline wix-scopes/src/index.js:13]: please don't throw, just log it and keep the extra object empty

File: wix-scopes/src/index.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      throw new Error(`unknown scope value ${scopeValue}`)
+    }
+  }
+
+  return scopesValues
+}
+
+exports.defineTags = function(dictionary) {
+  dictionary.defineTag(SCOPES_TAG_NAME, {
+    mustNotHaveValue : true,
```
Comment (nemni8) [inline wix-scopes/src/index.js:34]: are u sure those values are correct? can u please check in the code what they aim to achieve?

**Review by nemni8 — COMMENTED** [review]
goood job! had a few comments!

File: wix-scopes/src/index.js (hunk 3 above)
Comment (Galcarmi) [reply wix-scopes/src/index.js]: yep, they are correct, it is very clear in the docs, check it out if you want https://jsdoc.app/about-plugins.html

Comment (Galcarmi) [reply wix-scopes/src/index.js]: @nemni8

**Review by nemni8 — APPROVED** [review]
very nice work :) 
you are now an official docworks wizard 🧙

**Pull request merged**